docs/tutorials/intro.md

```diff
@@ -6,39 +6,43 @@ The library provides a layer of APIs for all software to enable internationaliza
 
 To use `ICU4X` in the Rust ecosystem one can either add dependencies on selected components, or add a dependency on the meta-crate `icu` which brings a reasonable selection of components in the most user-friendly configuration of features.
 
-In this tutorial we are going to start with the meta-crate and then introduce a customization.
+In this tutorial we are going to build up to writing an app that uses the `icu::datetime` component to format a date and time in a Japanese locale, covering various topics in the process.
 
 # 1. Requirements
 
-For this tutorial we assume the user has basic Rust knowledge. If acquiring it is necessary, [Rust Book](https://doc.rust-lang.org/book/) provides an excellent introduction.
-We also assume that the user is familiar with a terminal and have `git`, `rust` and `cargo` installed.
+For this tutorial we assume the user has basic Rust knowledge. If acquiring it is necessary, the [Rust Book](https://doc.rust-lang.org/book/) provides an excellent introduction.
+We also assume that the user is familiar with a terminal and have `rust` and `cargo` installed.
 
 To verify that, open a terminal and check that the results are similar to:
 
 ```
-user@host:~/projects/icu$ git --version
-git version 2.31.1
-user@host:~/projects/icu$ cargo --version
+$ cargo --version
 cargo 1.51.0 (43b129a20 2021-03-16)
 ```
 
-In this tutorial we are going to use a directory relative to the user's home directory `~/projects/icu/`. The `~` in the path indicates the relative location of the user home directory.
-Please create the directory structure necessary.
+In this tutorial we are going to use a directory relative to the user's home directory `~/projects/icu_tutorial/`. 
 
 # 2. Creating MyApp
 
 To initialize a binary application, navigate to the root directory of our project and initialize a new binary app called `myapp`:
 
 ```
-cd ~/projects/icu
-cargo init --bin myapp
+cd ~/projects/icu_tutorial
+cargo new --bin myapp
+cd myapp
 ```
 
-The result is a new directory `~/projects/icu/myapp` with a file `./src/main.rs` which is the main file for our application.
+The result is a new directory `~/projects/icu_tutorial/myapp` with a file `./src/main.rs` which is the main file for our application. For most of this tutorial we'll be working from the `myapp` folder.
```

Comment:
Nit: I've never been comfortable with the `icu` name since it conflicts with regular ICU, but I'm living with it on crates.io since it is a Rust ecosystem thing and we reached consensus long ago to go with that name. However, when creating any directories or other identifiers, I would really prefer if we would stick with `icu4x` as our primary identifier.